src/oracle.ts

```diff
@@ -63,24 +68,12 @@ export class Oracle {
         if(serialNumberGt(inception, proof.signature.data.inception)) {
             throw new OutdatedDataError(proof);
         }
-        const expired = serialNumberGt(this.now() / 1000, expiration + proof.signature.data.originalTTL);
```

Comment:
Are there any test cases which were passing previously but now fails due to this change?

Reply:
Yes, the one that's commented out until we get more data.